packages/ui/src/components/molecules/JoystickInteraction/types.ts

```diff
@@ -1,11 +1,10 @@
-import {
-  joystickArrowCompletedIcon,
-  joystickArrowIcon,
-  joystickArrowSelectedIcon,
-} from '../../../assets/images';
-import { ImageProps, IndicatorState } from '../../atoms';
+import { SvgProps } from '../../../assets';
+import { getDefaultTheme } from '../../../themes';
```

Comment:
Do not import theme from `themes` folder, use `useTheme` hook

Reply:
hooks cant be used outside FCs, what do you suggest